packages/react-core/src/components/Popover/Popover.tsx

```diff
@@ -261,7 +261,20 @@ export const Popover: React.FunctionComponent<PopoverProps> = ({
   showClose = true,
   distance = 25,
   // For every initial starting position, there are 3 escape positions
```

Comment:
Would this comment still be accurate? Originally the default loops back to `top, right, bottom` at the end, but this updated default doesn't.

Reply:
`hasAutoWidth` maybe isn't required, it's possible the fallback position defaults are enough for most use cases. I think for very small screen sizes the auto width will make the popper box smaller which is what the initial issue used as a workaround.

I can adjust the comment